README.md

```diff
@@ -1,2 +1,79 @@
-# dcos-e2e
-End to End Testing Harness for DC/OS
+# DC/OS End to End tests
+
+End to end tests are tests which require a DC/OS cluster to run against.
+Each test spins up at least one cluster, and has the choice of configuring this cluster as appropriate.
+For example, a test may require a cluster with a certain number of agents, or certain configuration options.
+
+The tests should be not be tied to the backend infrastructure.
+That is, they should pass against clusters on all supported infrastructures.
+The current implementation supports only a [DC/OS Docker](https://github.com/dcos/dcos-docker) backend.
+
+Usually tests are kept with the source code.
+However, this is a proof of concept.
+We plan to iterate on this repository and manually run tests.
+With that experience, we will choose where to put the test suite and whether it should be run on CI.
```

Comment:
The renderer doesn't render the new line characters.
This is purely for maintainability.
The idea comes from http://rhodesmill.org/brandon/2012/one-sentence-per-line/ and I personally have found that it is a useful practice.

The idea is that often a sentence is changed alone.
In normal prose style, the whole paragraph may be deleted and re-added in a `git` diff.
In this style, only the change is visible in the diff.

I have also found that the practice highlights to me, when writing, that a sentence is very long and so maybe it should be split up.